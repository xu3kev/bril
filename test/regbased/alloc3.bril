../regalloc/alloc3.out