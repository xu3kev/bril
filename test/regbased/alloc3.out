-6
9
3
#load: 14
#store: 14
